rpc/src/v1/types/trace.rs

```diff
@@ -223,6 +223,8 @@ pub struct Create {
 	gas: U256,
 	/// Initialization code
 	init: Bytes,
+	// Create Type
+	create_type: CreateType,
```

Comment:
I'd rather do `Option<CreateType>` instead and skip serializing if it's `None` cause it doesn't make sense for calls at all.